tests/cupy_tests/random_tests/test_distributions.py

```diff
@@ -1,9 +1,32 @@
 import unittest
 
+import numpy
+
+import cupy
+from cupy.random import distributions
 from cupy import testing
 
 
+@testing.parameterize(*testing.product({
+    'shape': [(3, 2)],
+    'loc_shape': [(), (3, 2)],
+    'scale_shape': [(), (3, 2)]
+})
```

Comment:
How about adding more cases in which some broadcastings occur (some of the tests you have implemented do it)?